soc/infineon/cat1a/Kconfig.defconfig

```diff
@@ -0,0 +1,6 @@
+# PSOC CAT1B Configuration
+
+# Copyright (c) 2024 Cypress Semiconductor Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+rsource "*/Kconfig.defconfig"
```

Comment:
looks like this is missing a SOC_SERIES_* guard?

Reply:
added